Cargo.toml

```diff
@@ -3,13 +3,13 @@
 name = "router"
 authors = ["Jonathan Reem <jonathan.reem@gmail.com>"]
 license = "MIT"
-version = "0.5.1"
+version = "0.6.0"
 description = "A router for the Iron framework."
 repository = "https://github.com/iron/router"
 documentation = "http://ironframework.io/doc/router/index.html"
```

Comment:
@untitaker should we point this to docs.rs while we're touching all the `Cargo.toml` files anyway?

Reply:
It redirects anyway, but yeah.